src/sql/workbench/services/profiler/browser/profilerService.ts

```diff
@@ -317,6 +318,18 @@ export class ProfilerService implements IProfilerService {
 		if (fileURIs?.length === 1) {
 			const fileURI = fileURIs[0];
 
+			try {
+				let fileSize = await (await fs.stat(fileURI.fsPath)).size;
```

Comment:
are both awaits needed?

Reply:
No the other await was unnecessary. Removing it.